loklak/loklak_webclient
# Wall creation step2

### IMP. Saved walls will not work. You'll need to delete all your saved walls.
Solves #330 and #305 
How to test:
* In step 2, you can add text in "All of these", "Any of these" and "none of these"
* check the query in the network tab if it is according to this: https://github.com/loklak/loklak_webclient/issues/330#issuecomment-122510004
